doc/modules/ROOT/pages/index.adoc

```diff
@@ -353,31 +353,28 @@ To install Service-Per-Pod Decorator Controller, you need to execute the followi
 
 [source, shell script]
 ----
-kubectl create configmap service-per-pod-hooks -n metacontroller --from-file=hooks
-kubectl apply -f service-per-pod.yaml
+kubectl apply -k service-per-pod
 ----
 
 This Decorator Controller automatically creates a service for each POD marked with the specific annotations.
 
 - Start Hazelcast cluster
 
+[source, yaml]
+.values.yaml
+----
+include::../../../../service-per-pod/values.yaml[]
+----
+
 [source, bash]
 ----
 helm repo add hazelcast https://hazelcast-charts.s3.amazonaws.com/
 helm repo update
-helm install hz-hazelcast \
-     --set service.type=LoadBalancer \
-     --set annotations.service-per-pod-label=statefulset.kubernetes.io/pod-name,annotations.service-per-pod-ports="5701:5701" \
-     hazelcast/hazelcast
+helm install hz-hazelcast -f service-per-pod/values.yaml hazelcast/hazelcast
 ----
 --
 ====
 
-[NOTE]
```

Comment:
Why did you remove this Note?

Reply:
It is not needed anymore, right? It fixes the situation mentioned in the Note section.

Reply:
Or do you want me to keep it for kubectl section?

Reply:
> Or do you want me to keep it for kubectl section?
`Note` section is not `Helm and Metacontroller` tab specific so we should keep it. I think we should provide more detailed version of it maybe?